misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -127,14 +127,14 @@ def get_report(coverage: Coverage) -> str:
     it gets any more complex.
     """
     try:
-        # Remove lines which are only covered in unit tests as sindicated by a
+        # Remove lines which are only covered in unit tests as indicated by a
         # negative line count
         for file, lines in coverage.items():
             with open(file, "r+") as f:
                 content = f.readlines()
                 f.seek(0)
                 for i, line in enumerate(content):
-                    if (lines.get(i + 1) or 0) < 0:
+                    if (lines.get(i + 1) or 0) >= 0:
```

Comment:
Philip reported:

> the coverage report lists commented-out lines as being covered by unit tests

This one is embarrassing. The comment above was correct, but I turned around the logic here later.